docs/proposals/mapper-support-video-streaming-data.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Mapper Support Video Streaming Data
+authors:
+- "@wbc6080"
+  approvers:
+  creation-date: 2024-3-5
+  last-updated: 2024-4-9
+  status: implementable
+---
+
+# Mapper Support Video Streaming Data
+
+## Motivation
+
+The data types currently collected by Mapper are discrete, such as `int`,`float` and other data types. 
+However, edge devices that collect data in the form of streaming data are also common, such as surveillance cameras in cities. 
+Mapper should enhance the management capabilities of streaming data devices.
+
+## Goals
+
+- Mapper can manage camera devices of edge nodes.
+- Mapper is able to obtain video streams from camera devices in edge nodes.
+- Add streaming data processing capabilities to the mapper-framework data plane.
+- Mapper is able to forward video streams from edge to the cloud. (still in planning, may support in future)
+
+
+## Design Details
+
+### Architecture
+
+<img src="../images/mapper/onvif-mapper.png">
```

Comment:
I cannot load the image

Reply:
Sorry, I submitted the old version of the document by mistake. Now the document has been corrected. @Shelley-BaoYue